addons/stock_landed_costs/wizard/purchase_config_settings.py

```diff
@@ -1,12 +1,10 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from openerp import api, models
-from openerp.tools.translate import _
+from odoo import api, models, _
 
 
 class PurchaseConfigSettings(models.TransientModel):
-    _name = 'purchase.config.settings'
     _inherit = 'purchase.config.settings'
 
     @api.onchange('group_costing_method')
```

Comment:
@WHPODOO 
yes i am sure that onchange gives a single record and this method is already migrated 
from this commit : https://github.com/odoo/odoo/commit/97dbfe54de330e8353f4cacd4dcb87a2c9431d97